app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
```

Comment:
I believe we're missing an XML declaration for this file. Please add.

Reply:
resolved